src/content/scrolling.ts

```diff
@@ -190,6 +202,7 @@ export async function recursiveScroll(
         ) {
             // Cache the node for next time and stop trying to scroll
             lastRecursiveScrolled = treeWalker.currentNode
+            lastFocused = document.activeElement // Invalidate the cache if the user changes focus
```

Comment:
It feels weird to do this here. The activeElement can't change while scrolling (unless the website detects scrolling and decides to change focus?) so I'd put this line 185 instead.

If you're setting `lastFocused` here to avoid to avoid `activeElement` changing under our feet, I think you need to also add it to the `lastRecursiveScrolled = treeWalker.currentNode` line below.

Reply:
I initially had it there. I moved it when click to change scroll wasn't working but I know realise that is a different bug.

Why don't we cache the element to scroll between presses of j/k?